setup.py

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env python3
 
-import fastentrypoints
```

Comment:
This is not unused. `fastentrypoints` uses monkeypatches easy_install.

Reply:
I dropped dropping fastentrypoints now and only remove `find_packages` now.